physics/module_sf_mynn.F90

```diff
@@ -1662,6 +1686,7 @@ SUBROUTINE SFCLAY1D_mynn(                                      &
     !----AND COMPUTE THE MOISTURE SCALE (or q*)
     !----------------------------------------------------
 
+    !tgs - should we have MOL and qstar separate for dry, icy and wet?
```

Comment:
We can probably make use out of both - especially for the negative qv problem. I think having a separate heat and moisture transfer coefficients may be the only way to fix that problem.